src/ogma/index.ts

```diff
@@ -334,12 +334,19 @@ export class LKOgma extends Ogma<LkNodeData, LkEdgeData> {
       this.store.clear();
     }
     this.initOgmaLinkuriousParser();
+    // we have to set the config manually because when instantiating Ogma, the config object is empty
+    // due to angular dependency injection (we cannot inject a global service asynchronously, and the config has not been loaded yet)
+    // when resetting Ogma, the config is reset to its initial value in Ogma which is an empty object.
+    // So when resetting Ogma we set the config again to have the correct config and not an empty one
+    this.setConfigOgma(this._configuration);
```

Comment:
This method is already called on `initOgmaLinkuriousParser()`. Your code is a bit weird... You pass this._configuration to a method that will set it to the same property. You're doing `this._configuration = this._configuration` here. I don't see how it can fix the problem...